sphinx/locale/tr/LC_MESSAGES/sphinx.po

```diff
@@ -3654,24 +3874,27 @@ msgstr "Arama Sonuçları"
 
 #: sphinx/themes/basic/static/searchtools.js:114
 msgid ""
-"Your search did not match any documents. Please make sure that all words are"
-" spelled correctly and that you've selected enough categories."
-msgstr "Arama sonucunda herhangi bir belge bulunamadı. Bütün kelimeleri doğru yazdığınızdan ve gerekli bütün kategorileri seçtiğinizden emin olun."
+"Your search did not match any documents. Please make sure that all words "
+"are spelled correctly and that you've selected enough categories."
+msgstr ""
+"Arama sonucunda herhangi bir belge bulunamadı. Bütün kelimeleri doğru "
+"yazdığınızdan ve gerekli bütün kategorileri seçtiğinizden emin olun."
 
 #: sphinx/themes/basic/static/searchtools.js:118
-msgid ""
-"Search finished, found ${resultCount} page(s) matching the search query."
-msgstr ""
+msgid "Search finished, found one page matching the search query."
+msgid_plural "Search finished, found ${resultCount} pages matching the search query."
+msgstr[0] ""
+msgstr[1] ""
```

Comment:
Ok, that's a good confirmation; previously `msgid_plural` hadn't appeared in any resource catalogs :+1: